api_unit_test.go

```diff
@@ -120,10 +120,10 @@ func TestPartSize(t *testing.T) {
 	if err != nil {
 		t.Fatal("Error: ", err)
 	}
-	if totalPartsCount != 79 {
+	if totalPartsCount != 40 {
 		t.Fatalf("Error: expecting total parts count of 79: got %v instead", totalPartsCount)
```

Comment:
`Error: expecting total parts count of 79` should be `Error: expecting total parts count of 40`